src/modules/endpoint/sagas/loadEvents.js

```diff
@@ -23,19 +23,25 @@ function * fetchEvents (city: string, language: string): Saga<?Array<EventModel>
 
 function * loadEvents (
   city: string, language: string, dataContainer: DataContainer, shouldUpdate: boolean): Saga<FetchMapType> {
-  if (dataContainer.eventsAvailable() && !shouldUpdate) {
-    console.debug('Using cached events')
-    return {}
-  }
+  let events: ?Array<EventModel>
```

Comment:
You could also have `const events = yield call(dataContainer.getEvents)` after the first if block (so we don't have `let`)

Reply:
Yes but this could take longer because work could possibly be done twice.

E.g. when getEvents does a sql query.